kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/build/revapi-config.json

```diff
@@ -22,11 +22,42 @@
       }
     }
   },
-
   "ignores": {
     "revapi": {
-    "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
-    "ignore": [
+      "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
+      "ignore": [
+        {
+          "code": "java.annotation.attributeValueChanged",
+          "old": "parameter javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.ui.ImageResource::getProcessInstanceImage(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Long, java.lang.String, java.lang.String, ===java.lang.String===)",
+          "new": "parameter javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.ui.ImageResource::getProcessInstanceImage(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Long, java.lang.String, java.lang.String, ===java.lang.String===, boolean)",
+          "annotationType": "javax.ws.rs.DefaultValue",
+          "annotation": "@javax.ws.rs.DefaultValue(\"#1e90ff\")",
+          "attribute": "value",
+          "oldValue": "\"#FF0000\"",
+          "newValue": "\"#1e90ff\"",
+          "package": "org.kie.server.remote.rest.jbpm.ui",
+          "classSimpleName": "ImageResource",
+          "methodName": "getProcessInstanceImage",
+          "parameterIndex": "5",
+          "elementKind": "parameter",
+          "justification": "[JBPM-9945] [GSS-RFE] [Runtime View] Being able to park process execution on failed script task and re-execute it when resuming process flow"
```

Comment:
@bxf12315 There are changes that shouldn't be part of this, in example this justification is out of date. This file needs to be regenerated from scratch